packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/VirtualizedTraceView.test.js

```diff
@@ -348,32 +385,14 @@ describe('<VirtualizedTraceViewImpl>', () => {
 
   describe('renderRow()', () => {
     it('renders a SpanBarRow when it is not a detail', () => {
-      instance = createTestInstance(mockProps);
-      const rowResult = instance.renderRow('some-key', {}, 1, {});
-
-      expect(rowResult.type).toBe('div');
-      expect(rowResult.props.className).toBe('VirtualizedTraceView--row');
-
-      const spanBarRow = rowResult.props.children;
-      expect(spanBarRow.type).toBe(SpanBarRow);
-      expect(spanBarRow.props.span).toBe(trace.spans[1]);
-      expect(spanBarRow.props.isChildrenExpanded).toBe(true);
-      expect(spanBarRow.props.isDetailExpanded).toBe(false);
+      render(<VirtualizedTraceViewImpl {...mockProps} />);
+      expect(screen.getByTestId('list-view')).toBeInTheDocument();
```

Comment:
this seems a lot weaker than the existing assertions.

Reply:
Are you saying that `render(<VirtualizedTraceViewImpl` somehow creates a shallow DOM that is different from prod? If the DOM is still full why can't we locate nested elements? And if the nested elements are actually an implementation detail of virtualized view then they probably should have their own FC that is testable.